databricks/koalas/generic.py

```diff
@@ -1193,7 +1193,7 @@ def abs(self):
 
     # TODO: by argument only support the grouping name and as_index only for now. Documentation
     # should be updated when it's supported.
-    def groupby(self, by, as_index: bool = True):
+    def groupby(self, by, axis=0, as_index: bool = True):
         """
         Group DataFrame or Series using a Series of columns.
 
```

Comment:
The parameter should be fixed too. Actually, why don't you try to implement the other axis? It wouldn't be impossible to do if we use pandas UDF from a cursory look. We have enough time before the next release currently.

Reply:
@HyukjinKwon  I'll be happy to do that.

Reply:
@HyukjinKwon  For now, I modified it to only supports axis = 0. (also docstring is fix)
I've looked into Pandas, but I don't have confidence in Koalas yet, 
so the implementation of other axis seems to take some time. 
Hmm.. Is it better I implement another axis as the next step after this PR?
 or do you want me to keep developing in this PR?

Reply:
Sure.